### PR TITLE
py-elasticsearch: Update to version 8.9.0; Add Python 311

### PR DESCRIPTION
**Pull request by essandess**
###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS x.y
Xcode x.y / Command Line Tools x.y.z

###### Verification 
Have you

- [ ] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [ ] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [ ] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [ ] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?
- [ ] checked that the Portfile's most important [variants](https://trac.macports.org/wiki/Variants) haven't been broken?

### CODE REVIEW

File: python/py-elasticsearch/Portfile [hunk 1]
```diff
@@ -4,7 +4,7 @@ PortSystem          1.0
 PortGroup           python 1.0
 
 name                py-elasticsearch
-version             8.3.1
+version             8.9.0
 platforms           darwin
```
Comment (reneeotten) [inline python/py-elasticsearch/Portfile:8]: it looks like that the port doesn't contain any compiled code and thus is independent of the architecture? Please verify, but if so one should likely set
```
platforms {darwin any}
supported_archs noarch
```

File: python/py-elasticsearch/Portfile [hunk 2]
```diff
@@ -14,11 +14,11 @@ long_description    ${description}
...
-                    size    303549
+checksums           rmd160  3f197083f5c342bd41aeab0e990f35f59c4320cc \
+                    sha256  d3367fc013e04fc7aad349a6de9fad1ee04fb6d627b0e7896aa505c12fde5e04 \
+                    size    313760
 
-python.versions     37 38 39 310
+python.versions     37 38 39 310 311
 
 if {${name} ne ${subport}} {
     depends_build-append \
```
Comment (reneeotten) [inline python/py-elasticsearch/Portfile:24]: upstream lists `install_requires = ["elastic-transport>=8,<9"]` - I don't see that here, but it appears that the build succeeds. Can you please check and clarify?

**Review by reneeotten — CHANGES_REQUESTED** [review]

File: python/py-elasticsearch/Portfile (hunk 1 above)
Comment (essandess) [reply python/py-elasticsearch/Portfile]: Thanks, done.

File: python/py-elasticsearch/Portfile (hunk 2 above)
Comment (essandess) [reply python/py-elasticsearch/Portfile:25]: Thanks, done.

**Pull request merged**